erc20-foundry/foundry.toml

```
@@ -0,0 +1,16 @@
+[profile.default]
+src = 'src'
+out = 'out'
+libs = ['lib']
+
+solc_version = '0.8.23'
+evm_version = 'shanghai'
+verifier_url = 'https://server-verify.hashscan.io'
+
+[rpc_endpoints]
+h_testnet = "https://testnet.hashio.io/api"
+h_mainnet = "https://mainnet.hashio.io/api"
+
+[etherscan]
+h_testnet= { url = "https://server-verify.hashscan.io", verifier= "sourcify", key="" }
```

```suggestion
h_testnet= { url="https://server-verify.hashscan.io", verifier="sourcify", key="" }
```
(consistency with whitespace)